### ISSUE TITLE
Allow user to indicate (open) license the original resource is shared under

### ISSUE THREAD

**Issue by dajbelshaw** [toolness/weblitmapper#28]
Give the user an option of choosing which Creative Commons license the original resource was shared under (if indeed it was). This would allow us to show openly-licensed resources at-a-glance when scanning the gallery.

(surfaced from https://etherpad.mozilla.org/weblitmapper-feedback)

**Comment by toolness** [comment]
Hmm, so this would be rockin, but isn't there also metadata in the form of HTTP headers or HTML markup that indicates this? If there is, then perhaps we could automatically fill-in this info via hubble! (see also #4)

**Comment by dajbelshaw** [comment]
Auto-filling would be awesome, but an 'advanced' optional field for the user would also be great. :-)

**Comment by toolness** [comment]
Yeah, that is a good idea--particularly for the resources (like a lot of my own) that say they're CC-licensed but don't have the machine-readable markup to say so.
